src/homework2/index.js

```diff
@@ -1,11 +1,20 @@
 import express from 'express';
 import userRouter from './routers/userRouter.js';
+import logRequest from './middlewares/logRequest.js';
+import logError from './middlewares/logError.js';
```

Comment:
It's quite common practice to create `index.js` file for the folders where we store exports from all the folder files. So the imports look like:
```
import { logRequest, logError } from './middlewares';
```